en/docs/design/api-policies/choreo-connect-policies/create-custom-cc-policy.md

```diff
@@ -17,17 +30,17 @@ Choreo Connect supports the following operations and the policies supporting the
             <li>headerName</li>
             <li>headerValue</li>
         </ul></td>
-    <td>Request</td>
-    <td>Set a header in the request flow.</td>
+    <td>Request Response</td>
+    <td>Set a header in the request flow or response flow.</td>
```

Comment:
Can we change this to `*Sets* a header in the request flow or response flow` ? If possible let's apply the verbs used in other rows also.